### ISSUE TITLE
Dry run of installation on MacOS/Linux

### ISSUE THREAD

**Issue by leonardt** [phanrahan/magmathon#8]
We should do a dry run of the installation process on both MacOS and Linux to verify everything is in order. In particular, we should test the `install_icestorm.py` script.

**Comment by leonardt** [comment]
I added a Dockerfile and published a docker image that I've tested on both MacOS and Windows. The steps are a bit clunky because we have to use a special setup to be able to pass USB devices through to the container, but it works and is portable. I'll plan to test the Docker image on linux as well.

I also have a Vagrant script that provisions a standard ubuntu VM that I've tested. There some wonkiness with programming the device (have to use sudo, can't figure out why this isn't working even with the udev rule added), but this should cover software/os support for most environments.

**Comment by leonardt** [comment]
Lot's of issues at the FB hackathon because users didn't have permissions for brew to write into `/usr/local/lib`. I suggest we drop the install script, and instead provide instructions for users to install the dependencies manually (through brew or apt). This way they can better process errors in the install process (rather than just trying to push the script all the way through to the end). The reasoning for not using a script is that there are too many variations in user setups (e.g. do they own /usr/local or not). So rather than try to cover all the cases, just have a list of instructions for users to follow and have them deal with any specific issues manually.

**Comment by rsetaluri** [comment]
I agree -- I think wrapping a few call's to `git clone` and `make` is overkill. On that note, if you go to: https://github.com/phanrahan/magmathon#icestorm-tools, I think we can make it very explicit. Like:

1. Download and install arachne-pnr: `git clone <...>` then `cd arachne-pnr` and `make`
2. Download and install ...
